README.md

```diff
@@ -1,25 +1,26 @@
-# Beacon SDK Samples for Android
+# Beacon Android SDK Samples
 
-This repo contains 3 sample apps that implement the Beacon Android SDK. 
+This repo contains 2 sample apps that implement the Beacon Android SDK.
 
-* [Sample Kotlin](https://github.com/helpscout/beacon-android-sdk-sample/tree/master/sample-kotlin) - demos Login and Logout with the Beacon UI.
+* [Sample Kotlin](https://github.com/helpscout/beacon-android-sdk-sample/tree/master/sample-kotlin) - demos how to install, initialise, and opening your Help Scout Beacon.
+.
 * [Sample Customisation](https://github.com/helpscout/beacon-android-sdk-sample/tree/master/sample-customisation) - illustrates how you can override the String and Colors of the Beacon UI and shows how to use Beacons in Secure mode.
 
+## Beacon SDK version [![Download Beacon UI](https://api.bintray.com/packages/helpscout/beacon/beacon-ui/images/download.svg) ](https://bintray.com/helpscout/beacon/beacon-ui/_latestVersion)
 
-## Beacon SDK version [![Download Beacon UI](https://api.bintray.com/packages/helpscout/beacon/beacon-ui/images/download.svg) ](https://bintray.com/helpscout/beacon/beacon-ui/_latestVersion) 
-
-[Change Log](https://github.com/helpscout/beacon-android-sdk-sample/blob/master/CHANGELOG.md)
+[Release notes and change Log](https://github.com/helpscout/beacon-android-sdk-sample/blob/master/CHANGELOG.md)
```

Comment:
🔍  Use `changelog` instead? I feel like I've seen more "changelog" than "change log". In any case, "Log" should be lowercased.
```suggestion
[Release notes and changelog](https://github.com/helpscout/beacon-android-sdk-sample/blob/master/CHANGELOG.md)
```